packages/url/src/test/index.test.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
The `/**` the above comment is on is for a file header comment @gziolo 

We can add these paths in `package.json` after merging this PR.

As to what's popular in the WordPress world for this, not much prior art, so "best practices" is what we should aim for